### ISSUE TITLE
Bridge tx debited but not credited in my Metamask wallet

### ISSUE THREAD

**Issue by HoTandy** [FuelLabs/fuel-bridge#131]
Yesterday I made a Bridge transaction from my Fuel wallet to a a Sepolia Wallet I set up in Metamask.

Fuel wallet version: `Version: 0.15.2`

![image](https://github.com/FuelLabs/fuel-bridge/assets/114662397/6c25f403-2a48-43b2-adcf-2b93fd444d2c)

I clicked away from the bridge when it was in the state in screenshot above.

The transaction ID is: [https://app.fuel.network/tx/0xcae664b7a7f8f4cd19ed3c4232b3a1d52fcdf798b36e99e9a29ff0f7235e7c0c/simple](https://app.fuel.network/tx/0xcae664b7a7f8f4cd19ed3c4232b3a1d52fcdf798b36e99e9a29ff0f7235e7c0c/simple)

I see that the transaction has taken place in my Fuel wallet:

<img width="343" alt="image" src="https://github.com/FuelLabs/fuel-bridge/assets/114662397/c62a3faa-e63d-435b-8694-01da6730cd3e">

from: `fuel1dedeagaksypljjga2yy95dkxl2lk555dkujmu8qmrgerpxypvdvqxsl2ut` (copied from fuel wallet)
to: `0x000000000000000000000000940aa7516018d954736c2ab1628337666f302fd8` (copied from fuel wallet)
and: `0x940AA7516018D954736C2ab1628337666F302fd8` (copied from Metamask wallet)

<img width="343" alt="image" src="https://github.com/FuelLabs/fuel-bridge/assets/114662397/9bde3c8b-4ef9-4e9f-ae3c-47117f3a11f9">

Only when I click to History do I see that I have to confirm the transaction.

<img width="449" alt="image" src="https://github.com/FuelLabs/fuel-bridge/assets/114662397/f3601cde-cf0e-47d3-ab44-5910e88eb9dd">

<img width="449" alt="image" src="https://github.com/FuelLabs/fuel-bridge/assets/114662397/cd6c2a6e-9079-409a-81a9-c0d4b674cd1b">

When I click confirm Metamask pops up as if the Bridge is asking Metamask to sign the transaction.

<img width="1131" alt="image" src="https://github.com/FuelLabs/fuel-bridge/assets/114662397/c1e15ad3-1693-4263-9074-d2fd796e40fe">

but it cannot as there's not enough ETH in the wallet to do this.

but should this be happening at all?  Surely the receiving wallet should not need to sign a transaction, or confirm an incoming transaction or have to pay a fee for an incoming transaction.

Is this a bug?  Please let me know if I'm doing something wrong.  Maybe the Bridge is opening Metamask when it should be opening the Fuel wallet extension.  Or is this how it works with this kind of smart contract?

**Comment by LuizAsFight** [comment]
this is how it works. the last step of withdraw is relay the message on ETH side to complete the transaction. that can only be done by you using your signature.

After you complete this transaction, you will see the balance in your metamask. Did you try to complete? you can faucet some Sepolia ETH to your  metamask wallet here: https://www.alchemy.com/faucets/ethereum-sepolia

**Comment by HoTandy** [comment]
Excellent, thanks for the additional information, I'll some Sepolia ETH and see if the transaction completes, if it does, ....we should be able to close this bug.

**Comment by LuizAsFight** [comment]
also, in the bridge screen while starting a WITHDRAW ERC-20 operation, you will see an alert saying:
```
You don't have any ETH on Fuel to pay for gas. We recommend you bridge some ETH before you bridge any other assets.```
                
On this way we prevent users of starting the operation not having the necessary funds to finish it in ETH side.

The same alert could be added to prevent withdrawing ETH with not having funds on eth side also

**Comment by HoTandy** [comment]
Hi Luiz, 3 days ago there was some kind of network error with the Alchemy faucet, I finally managed to get a hold of some Sepolia ETH on my (Sepolia) Metamask.  Then I was able to confirm the transaction with Metamask.  So apart from Fuel TS (or JS) consuming a lot of Chrome resources and the fans lightning up on my machine [https://github.com/FuelLabs/fuel-bridge/issues/133](https://github.com/FuelLabs/fuel-bridge/issues/133), the whole process was smooth in the end.

Thanks a lot for your support! I'll close this (non-)"bug" :-)
